microsoft/testsuites/performance/storageperf.py

```diff
@@ -241,9 +241,11 @@ def _perf_nfs(
         # calculate the number of jobs to run. The max number of jobs is equal
         # to the core count of the node.
         num_jobs = []
+        iodepth_iter = start_iodepth
         core_count = client_node.tools[Lscpu].get_core_count()
-        for i in range(start_iodepth, max_iodepth + 1):
-            num_jobs.append(min(2 ** (i - 1), core_count))
+        while iodepth_iter <= max_iodepth:
+            num_jobs.append(min(iodepth_iter, core_count))
```

Comment:
Add comments to explain the purpose of this num_jobs, why it's created like this way.

Reply:
There is a comment present above: "The max number of jobs is equal to the core count of the node.". Is there something specific that you want to add?

Reply:
It's not clear. The core count of this node is a number like "32", but how it comes out an array? And what the "job" is for?

Reply:
Sure! I will add an example with more description to make it explicit